src/lib/bundleUtils.ts

```diff
@@ -99,7 +101,7 @@ export const getUnlockBundlesFromOvalAddresses = async (
       targetBlock,
       ovalAddress,
       req,
-      false,
+      false, // Do not simulate
```

Comment:
What does this comment mean? If we're not simulating, wouldn't this be true...?

Reply:
[Simulate has a default value of true](https://github.com/UMAprotocol/oval-node/blob/75e2276031ba68ab879dfd7b26129a088acb784b/src/lib/bundleUtils.ts#L56), but here we are not simulating. I added the comment because now we have a second optional boolean argument, `shouldRecordWalletUsage`, so it’s easier to read.